src/python/pants/jvm/goals/lockfile_test.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
```

Comment:
These tests are very similar to `coursier_integration_test.py`, but they're more direct. We don't run a goal_rule anymore, and they instead test the rules directly.